src/test/java/academy/softserve/edu/enums/CustomerTypes.java

```diff
@@ -0,0 +1,21 @@
+package academy.softserve.edu.enums;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public enum CustomerTypes {
+
+    STANDART,
+    SILVER,
+    GOLD,
+    PLATINUM;
+
+    public static final String getCustomerTypeNameByReference(final int customerTypeReference) {
+
```

Comment:
no need for brackets (not sure, re-check):
.filter(customerTypes -> customerTypes.ordinal() + 1 == customerTypeReference)

no need to map, you already have strings

.filter(customerTypes -> (customerTypes.ordinal() + 1 == customerTypeReference)
.findFirst()
.get()